modules/rosa-list-objects.adoc

```diff
@@ -176,6 +175,42 @@ List all API and ingress endpoints for a cluster named `mycluster`:
 $ rosa list ingresses --cluster=mycluster
 ----
 
+== list instance-types
+
+List all of the available instance types for use with {product-title}. Availability is based on the account's AWS quota.
+
+.Syntax
+[source,terminal]
+----
+$ rosa list instance-types [arguments]
+----
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--output
+|The output format. Allowed formats are [json \| yaml].
+
+|--profile
+|Specifies an AWS profile (string) from your credentials file.
+|===
+
+.Example
+List all instance types:
+
+[source,terminal]
+----
+$ rosa list instance-types
+----
+
 [id="rosa-list-machinepools_{context}"]
 == list machinepools
 
```

Comment:
So this isn't related to this PR at all, but when checking the rendered build, I noticed that one of the section headings wasn't rendering properly. It's literally showing this in the preview:

```
 [id="rosa-delete-upgrade-cluster_{context}"]
== delete upgrade
```
If that's something you could look into fixing in a future PR? I'm guessing it just accidentally has an extra space or two in front of it.